src/main/scala/com/databricks/spark/xml/util/XSDToSchema.scala

```diff
@@ -39,8 +39,11 @@ object XSDToSchema {
    */
   @Experimental
   def read(xsdFile: File): StructType = {
-    val xmlSchema = new XmlSchemaCollection().read(
+    val xmlSchemaCollection = new XmlSchemaCollection()
+    xmlSchemaCollection.setBaseUri(xsdFile.getParent)
```

Comment:
Is this not the default? I'm fine with it if it helps.

Reply:
No, this is the whole point of this PR :)

Reply:
OK weird I would have thought it assumes that somehow. The change is OK.